webserver/webserver/src/main/java/io/helidon/webserver/WebServer.java

```diff
@@ -751,6 +751,17 @@ public Builder workersCount(int workers) {
             return this;
         }
 
+        /**
+         * Configure whether to enable content negotiation for compression.
+         *
+         * @param value compression flag
+         * @return updated builder instance
+         */
+        public Builder enableCompression(boolean value) {
```

Comment:
This method should be annotated with `@Override` and the javadoc is not necessary. If the javadoc is there, please add "negotiation for compression for the default socket"